### ISSUE TITLE
Build Failures

### ISSUE THREAD

**Issue by mav8557** [GJDuck/e9afl#4]
Hello,

I would like to start using this program but ran into several build errors - would it be possible to publish a formal list of dependencies, if any?

**Comment by GJDuck** [comment]
Did you use the `build.sh` script?  What are the build errors?

There are also binary releases: https://github.com/GJDuck/e9afl/releases

**Comment by mav8557** [comment]
I did use the build script, but because it doesn't check for errors between steps it printed out several until it couldn't continue. They first start appearing after attempting to build e9patch and e9tool, mostly because of some missing headers and definitions but eventually due to other reasons. I've included the output from the script on my computer, this was run from an Ubuntu 20.04 VM.

https://pastebin.com/u0XE4U5N

Also, the compiled binary worked, so in the meantime I will use it. Thank you for releasing this project.
